cmd/root.go

```diff
@@ -46,13 +45,10 @@ func Execute() {
 
 func init() {
 	persistent := rootCmd.PersistentFlags()
-	persistent.BoolP("disable_slack", "d", false, "Disable Slack alerts.")
-
 	persistent.StringP("config", "c", "config.toml", "Config file path.")
-
+	persistent.StringSliceP("reporters", "r", []string{"slack"}, "Specify a list of reporters for reporting vulnerabilities.")
 	persistent.BoolP("quiet", "q", false, "Suppress all console output. (Mutually exclusive with 'verbose'.)")
 	persistent.CountP("verbose", "v", "More verbose output. Specifying multiple times increases verbosity. (Mutually exclusive with 'quiet'.)")
 
-	_ = viper.BindPFlags(persistent)
```

Comment:
For some reason, this wasn't binding the cobra flags to viper...

Reply:
Most likely it's a timing issue, caused by the fact that this is in `init()` (making this a side-effect of the package loading... which is somewhat undesirable in itself) and the fact the the config defaults are only loaded much later, when `GetUserConfig()` is called. I will take a stab at fixing this today!